src/Viz.js

```diff
@@ -348,7 +348,7 @@ export default class Viz extends BaseClass {
       : this._groupBy.length - 1;
 
     // Returns the current unique ID for a data point, coerced to a String.
-    this._id = (d, i) => `${accessorFetch(this._groupBy[this._drawDepth], d, i)}`;
+    this._id = (d, i) => typeof accessorFetch(this._groupBy[this._drawDepth], d, i) === "number" ? `${accessorFetch(this._groupBy[this._drawDepth], d, i)}` : accessorFetch(this._groupBy[this._drawDepth], d, i);
```

Comment:
Can you rewrite this into a multi-line function to store `accessorFetch(this._groupBy[this._drawDepth], d, i)` as a local variable? It will make the ternary line a lot shorter and easier to read, as well as reducing running the `accessorFetch` function from 2 times to 1 (in your code, it is run to evaluate the `typeof`, and then run again in the else/then).

Since these types of functions are potentially run many times on long data arrays throughout the viz render cycle by different components, doubling the function invocation could give us some unexpected slow-downs. 😅

Reply:
You're right! I just pushed a commit to change it 💪!